client/src/components/LightBox/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SwipeRightImg from '../../images/SwipeRightImg'
+import SwipeLeftImg from '../../images/SwipeLeftImg'
+import { Layer, Text } from './styled'
+
+const categoryMap = {
+  nuxSwipeLeft: {
+    texts: ['SWIPE LEFT', 'TO PASS ON', 'THIS FILM'],
+    img: <SwipeLeftImg />,
+  },
+  nuxSwipeRight: {
+    texts: ['SWIPE RIGHT', 'TO CHOOSE', 'THIS FILM'],
+    img: <SwipeRightImg />,
+  },
+  nuxSwipeDown: 'nuxSwipeDown',
+}
+
+export const categories = Object.keys(categoryMap).reduce((acc, curr) => {
```

Comment:
// Returns an object generated from categoryMap keys where
// the keys and values are the same
// example return value
// {
//  nuxSwipeLeft: "nuxSwipeLeft"
//  nuxSwipeRight: "nuxSwipeRight"
// }